### PR TITLE
ci: macOS Runtime tests and Snapshots

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Feature
- Build or CI related changes

## What is the current behavior?

- Command-line arguments are not passed to macOS app's `LaunchActivatedEventArgs`
- Multiple arguments are separated by `;`
- CI does not run runtime tests and snapshots for macOS 

## What is the new behavior?

- Command-line arguments are passed to macOS app's `LaunchActivatedEventArgs`
- Multiple arguments are separated by ` ` to match UWP/WinUI **this applies to Skia GTK too!**
- CI does run runtime tests and snapshots for macOS

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: build/ci/.azure-devops-macos.yml [hunk 1]
```diff
@@ -53,7 +46,7 @@ jobs:
       msbuildLocationMethod: version
       msbuildVersion: latest
       msbuildArchitecture: x86
-      msbuildArguments: /r /p:Configuration=Release /detailedsummary /bl:$(build.artifactstagingdirectory)/build-macos.binlog
+      msbuildArguments: /r /p:Configuration=Debug /p:EnableCodeSigning=false /p:UseHardenedRuntime=false /detailedsummary /bl:$(build.artifactstagingdirectory)/build-macos.binlog
```
Comment (spouliot) [inline build/ci/.azure-devops-macos.yml:49]: That (both flags) looks problematic.

That's the kind of changes that will build just fine but will create issues at runtime.

**Review by spouliot — COMMENTED** [review]
@MartinZikmund  Just had a quick look and very tempted to blame code signing :)

However the azure logs are not available anymore, so the exact reason why nothing was reported (it's green) remains a mystery.

I'll try to fix the merge conflicts asap to see the logs.

**Comment by spouliot** [comment]
@MartinZikmund looks like the execution startup errors are gone... but nothing is being executed (based on the time macOS tests took), nor was anything logged. Is there a way to have more verbose output from the tests ?

**Comment by MartinZikmund** [comment]
Added by #7830, closing

**Pull request closed without merging**